includes/wp-admin/table/class-followers.php

```diff
@@ -392,6 +405,28 @@ protected function handle_row_actions( $item, $column_name, $primary ) {
 			),
 		);
 
+		if ( \boolval( \get_option( 'activitypub_following_ui', '0' ) ) ) {
+			if ( Following::check_status( $this->user_id, $item['id'] ) ) {
+				$actions['follow'] = \__( 'You follow each other', 'activitypub' );
```

Comment:
Let's not show that? If they already follow back, then let's just not have this action available.